package.json

```diff
@@ -61,6 +62,10 @@
 		"eslint-plugin-import": "^2.25.4",
 		"file-loader": "^6.2.0",
 		"h2o2": "^8.2.0",
+		"jsdoc": "^3.6.10",
+		"jsdoc-advanced-types-plugin": "github:tomalec/jsdoc-advanced-types-plugin#add/return-support",
+		"jsdoc-plugin-intersection": "^1.0.4",
+		"jsdoc-plugin-typescript": "^2.0.6",
```

Comment:
❓  Why do we need TS support?

Reply:
Because we use TS notation in our code comments, like  imported types `{import('data/actions').ShippingRate}`